tests/testthat/test-municipality_codes.R

```diff
@@ -0,0 +1,7 @@
+test_that("correctly reads municipality codes and respective names", {
+  data <- read_municipality()
+  expect_identical(colnames(data), c("Gemeentecode", "GemeentecodeGM",
+                                     "Gemeentenaam", "Provinciecode", "ProvinciecodePV",
+                                     "Provincienaam"))
+  expect_identical(data$Gemeentenaam[which(data$GemeentecodeGM == "GM1684")], "Cuijk")
+})
```

Comment:
The data retrieval function of read_municipality should be mocked. We don't want to put an unnecessary load on CBS servers, plus it will not run if you are offline.